components/opacitycheckerboard/stories/opacitycheckerboard.stories.js

```diff
@@ -5,7 +5,7 @@ import { styleMap } from "lit/directives/style-map.js";
 import { Template } from "./template";
 
 export default {
-	title: "Components/Opacity checkerboard",
+	title: "Utilities/Opacity checkerboard",
```

Comment:
This is a good clarification but let's hold off creating new headings yet. I'd like to go through our whole list of components and determine what others can fall under this same category.

Reply:
reverted for now! 

I agree that it would be useful to look at all components and see what categories make sense.